spec/package_spec.rb

```diff
@@ -300,7 +300,7 @@ def build_app_source_with_files(files)
 
         it 'includes zh-cn in translations' do
           expected_translations = JSON.parse(File.read('spec/translations/zh-cn.json'))
-          expect(package.send(:translations)['zh-cn'].except('custom1')).to eq(expected_translations)
+          expect(I18n::Utils.except(package.send(:translations)['zh-cn'], 'custom1')).to eq(expected_translations)
```

Comment:
Hash refinements are removed in recents i18n versions (https://github.com/ruby-i18n/i18n/pull/573).